src/shell/shell.cpp

```diff
@@ -1258,6 +1260,31 @@ void SHELL_Init() {
 	        " Volume in drive %c is %s\n"
 	        " Volume Serial Number is %04X-%04X\n"
 	        "\n");
+	MSG_Add("SHELL_CMD_MOVE_HELP",
+	        "Moves files and renames files and directories\n");
+	MSG_Add("SHELL_CMD_MOVE_HELP_LONG",
+	        "Usage:\n"
+	        "To move one or more files:\n"
+	        "  [color=green]move[reset] [color=cyan][drive:][path]filename1[,...] destination[reset]\n"
+	        "\n"
+	        "To rename a directory:\n"
+	        "  [color=green]move[reset] [color=cyan][drive:][path]dirname1 dirname2[reset]\n"
```

Comment:
`[color=green]move[reset] [color=cyan]DIRECTORY1[reset] [color=cyan]DIRECTORY2[reset]\n` - for uniformity with `mkdir` command help.

Reply:
Fixed